### ISSUE TITLE
Wrong UTC time setting when starting Stellarium

### ISSUE THREAD

**Issue by pacastro** [Stellarium/stellarium#530]
### Expected Behaviour
Correct UTC time setting for selected location in Stellarium each time app is started.

### Actual Behaviour
UTC time setting shows UTC-4:41 each time the app is started, current correct time for selected location is UTC -3:00.
<img width="1127" alt="screen shot 2018-10-28 at 1 55 04 pm" src="https://user-images.githubusercontent.com/35240993/47619128-03f59b00-daba-11e8-9c3a-51561cfea26b.png">

### Steps to reproduce
Start or restart app, it will show UTC-4:41 with  location set to Antofagasta. If you open Location Window, the time goes immediately to the correct UTC-3:00 without having to correct anything there.

### System
* Stellarium version: Stellarium 0.90.0.16375
* Operating system: OSX 10.14
* Graphics Card: Intel HD Graphics 5300, 2.1 INTEL-12.0.34

### Logfile 
[log.txt](https://github.com/Stellarium/stellarium/files/2522987/log.txt)

**Comment by gzotti** [comment]
It seems you have stored your location with timezone set to  Local Mean Solar Time (LMST). Set your location to Antofagasta from the location list, make sure you have the correct timezone (or just disable "Select Timezone"), and re-store this location.

**Comment by pacastro** [comment]
Ok, now I think I got to reproduce this as a bug....

I have settings as described in my last post: location set to Antofagasta, "use custom time zone" unchecked... so it goes to the correct default UTC-3:00. so far so good. works ok. "use current location as default" Checked, so this location is the default one.

Now, if I go to Location Window and select on the map any random location, it shows the coordinates for this location and the time zone goes to LMST, but only on the Location Window.... 

First bug: Stellarium lower info tab show the new coordinates OK, but the time shown stays in the previous default setting of UTC-3:00, it should have gone to the UTC-4:41 of the LMST seted in location window by selecting in the map.

Second bug: Now if I restart the app, it starts showing on the lower info tab the default location (in my case Antofagasta), but the time zone shown is LMST UST-4:41. 

So. should it have stayed in the location it was when the app was closed?? (random selected from map), in that case the time zone is ok, but location is wrong. Or should it start in the default location??? in that case location is OK but time zone was not updated to the default one (UTC -3:00, just by opening Location Window the time zone goes to the default location one).
